src/entities/token.ts

```diff
@@ -56,28 +56,49 @@ export function currencyEquals(currencyA: Currency, currencyB: Currency): boolea
   }
 }
 
-export const WETH = {
-  [ChainId.MAINNET]: new Token(
-    ChainId.MAINNET,
-    '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
-    18,
-    'WETH',
-    'Wrapped Ether'
-  ),
-  [ChainId.ROPSTEN]: new Token(
-    ChainId.ROPSTEN,
-    '0xc778417E063141139Fce010982780140Aa0cD5Ab',
-    18,
-    'WETH',
-    'Wrapped Ether'
-  ),
-  [ChainId.RINKEBY]: new Token(
-    ChainId.RINKEBY,
-    '0xc778417E063141139Fce010982780140Aa0cD5Ab',
-    18,
-    'WETH',
-    'Wrapped Ether'
-  ),
-  [ChainId.GÖRLI]: new Token(ChainId.GÖRLI, '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6', 18, 'WETH', 'Wrapped Ether'),
-  [ChainId.KOVAN]: new Token(ChainId.KOVAN, '0xd0A1E359811322d97991E03f863a0C30C2cF029C', 18, 'WETH', 'Wrapped Ether')
+export const WETH_MAINNET = new Token(
+  ChainId.MAINNET,
+  '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+  18,
+  'WETH',
+  'Wrapped Ether'
+)
+
+export const WETH_ROPSTEN = new Token(
+  ChainId.ROPSTEN,
+  '0xc778417E063141139Fce010982780140Aa0cD5Ab',
+  18,
+  'WETH',
+  'Wrapped Ether'
+)
+
+export const WETH_RINKEBY = new Token(
+  ChainId.RINKEBY,
+  '0xc778417E063141139Fce010982780140Aa0cD5Ab',
+  18,
+  'WETH',
+  'Wrapped Ether'
+)
+
+export const WETH_GÖRLI = new Token(
+  ChainId.GÖRLI,
+  '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6',
+  18,
+  'WETH',
+  'Wrapped Ether'
+)
+export const WETH_KOVAN = new Token(
+  ChainId.KOVAN,
+  '0xd0A1E359811322d97991E03f863a0C30C2cF029C',
+  18,
+  'WETH',
+  'Wrapped Ether'
+)
+
+export const WETH: Partial<Record<ChainId, Token>> = {
```

Comment:
why's this Partial?

Reply:
I reverted this change, it was not good.